### ISSUE TITLE
Time for new gem release?

### ISSUE THREAD

**Issue by bootstraponline** [gmailgem/gmail#181]
There have been a few bug fixes so it'd be nice to make a new gem release.

**Comment by johnnyshields** [comment]
@bootstraponline please go ahead. Be sure to update changelog

**Comment by bootstraponline** [comment]
I'm not a gem owner so RubyGems won't allow me to publish gem releases.

```
$ gem owner gmail
Owners for gem: gmail
- jeff
- johnny.shields
- alex
- justin
- kriss.kowalik
```

**Comment by johnnyshields** [comment]
Just added you, please check now.

**Comment by bootstraponline** [comment]
It worked. I'll update the change log and release 0.6.0 later this week.

**Comment by bootstraponline** [comment]
I've learned my lesson about giving time estimates. I [updated the changelog](https://github.com/gmailgem/gmail/blob/master/CHANGELOG.md) and [released 0.6.0.](https://rubygems.org/gems/gmail)
